hack/after-build/run-codegen.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
for consistency - can you call this file "update-codegen.sh"

[it's strange to have update-codegen calling after-build/run-codegen]